src/features/Order/Checkout/utils/splitDate.ts

```diff
@@ -0,0 +1,3 @@
+export const splitDate = (date: string) => {
+  return date.split('-')[0] + `년 ` + date.split('-')[1] + `월 ` + date.split('-')[2] + `일`;
+};
```

Comment:
쿠폰 목록 조회 API 응답값은 expirationDate 필드의 값을 ISO 형식으로 내려주고 있으므로 문자열을 직접 파싱하기 보다는 Date 내장 메서드를 사용할 수 있을 것 같습니다. 

```suggestion
export const formatDate = (date: string) => {
  const dateObj = new Date(date);

  return `${dateObj.getFullYear()}년 ${dateObj.getMonth() + 1}월 ${dateObj.getDate()}일`;
};
```

Reply:
또한 함수의 이름 또한 'formatDate' 가 더 적절하다고 생각이 드는데 세라 생각은 어떠실까요?

Reply:
넴 Date 내장 메서드를 사용하는 방식이 훨씬 직관적이고 견고해 보여요! 단순히 split으로 자르는 방식보다 어떤 값이 연/월/일인지 더 명확하게 드러나고, 다양한 날짜 포맷에도 대응 가능해서 유지보수에도 유리할 것 같아요! 수정 진행했습니다!

[refactor: splitDate함수명 formatDate 로 변경, Date 내장 메서드 사용](https://github.com/woowacourse/react-shopping-cart/pull/382/commits/49c9e1f441514ebe51762712007894a9024af076)